ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install.cmake.in

```diff
@@ -30,7 +30,7 @@ function(ament_cmake_symlink_install_directory)
   endif()
 
   # default pattern to include
-  if("${ARG_PATTERN} " STREQUAL " ")
+  if(NOT ARG_PATTERN)
```

Comment:
Isn't the current check slightly stricter than the proposed conditional? Wouldn't this now pass if `ARG_PATTERN` is `OFF`, whereas it did not before?